dsgrid/config/dataset_config.py

```diff
@@ -452,6 +485,11 @@ def add_trivial_dimensions(self, load_data_lookup):
                 load_data_lookup = load_data_lookup.withColumn(col, F.lit(val))
         return load_data_lookup
 
+    def remove_trivial_dimensions(self, load_data_lookup):
```

Comment:
The dataframe passed in is not necessarily a load_data_lookup, right? More generic name?